### PR TITLE
fix gce.conf multi-value parameter processing

### LINKED ISSUE #56833: GCE cloud provider AlphaFeatureGate can't handle more then 1 feature
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line: 
>
/kind bug
> /kind feature

**What happened**:
If the `alpha-feature` field in `/etc/gce.conf` contains multiple entries, e.g. `alpha-features = NetworkTiers,DiskAlphaAPI,NetworkEndpointGroup`, none of them are enabled.

Log message:
```
$ cat /var/log/kube-controller-manager.log | grep -i "alpha feature "
E1205 04:26:39.285851       5 gce.go:263] Encountered error for creating alpha feature gate: alpha feature "Network
Tiers,DiskAlphaAPI,NetworkEndpointGroup" is not supported.
```

**What you expected to happen**:
Each of the comma-separated entries should be enabled.

**How to reproduce it (as minimally and precisely as possible)**:
Read above.

**Anything else we need to know?**:
Root cause appears to be the value of `configFile.Global.AlphaFeatures` is  not split before being used in https://github.com/kubernetes/kubernetes/blob/master/pkg/cloudprovider/providers/gce/gce.go#L275

CC @freehan @yujuhong  

**Environment**:
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

### PR DESCRIPTION
**Pull request by freehan**
fixes: https://github.com/kubernetes/kubernetes/issues/56833

alternative to: https://github.com/kubernetes/kubernetes/pull/56857

```release-note
NONE
```

### CODE REVIEW

File: cluster/gce/gci/configure-helper.sh [hunk 1]
```diff
@@ -585,14 +585,18 @@ EOF
   if [[ -n "${NODE_INSTANCE_PREFIX:-}" ]]; then
     use_cloud_config="true"
     if [[ -n "${NODE_TAGS:-}" ]]; then
-      local -r node_tags="${NODE_TAGS}"
+      IFS=',' read -r -a node_tags <<< ${NODE_TAGS}
```
Comment (bowei) [inline cluster/gce/gci/configure-helper.sh]: # Split node_tags into an array by comma.

File: cluster/gce/gci/configure-helper.sh [hunk 2]
```diff
@@ -602,9 +606,12 @@ EOF
   fi
   if [[ -n "${GCE_ALPHA_FEATURES:-}" ]]; then
     use_cloud_config="true"
-    cat <<EOF >>/etc/gce.conf
-alpha-features = ${GCE_ALPHA_FEATURES}
+    IFS=',' read -r -a alpha_features <<< ${GCE_ALPHA_FEATURES}
```
Comment (bowei) [inline cluster/gce/gci/configure-helper.sh]: Add comment (this is super obscure bashism)

**Comment by saad-ali** [comment]
/lgtm

This must be cherry picked back to 1.8

**Pull request merged**